Sources/XcodeGenKit/SettingsBuilder.swift

```diff
@@ -67,14 +67,22 @@ extension ProjectSpec {
                     buildSettings += configFile.flattenedBuildSettings()
                 }
             }
-            buildSettings += getProjectBuildSettings(config: config)
+            for (k, v) in getProjectBuildSettings(config: config) {
```

Comment:
Once the replacement logic is moved into getProjectBuildSettings, these can be removed